index.js

```diff
@@ -53,7 +53,8 @@ FixtureCreator.prototype.createFixture = function(feedName, params) {
             newFeedDefer.resolve(json);
         });
     } else {
-        newFeedDefer.resolve(feed)
+        cloneFeed = JSON.parse(JSON.stringify(feed));
```

Comment:
cloneFeed is never defined man, it is everywherrre.

Much global
Very window
Wow